content/docs/command-reference/fetch.md

```diff
@@ -100,9 +100,9 @@ specific one is given with `--remote`.
   the workspace. Note that both options can be combined, for example using the
   `-aT` flag.
 
-- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git commits
-  as well as the workspace. This downloads tracked data for the entire commit
-  history of the project.
+- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ your Git
```

Comment:
hmm ... it can be interpreted like only my commits (authored by me)?

Reply:
Ah good point. Idk how to best express this... 🤔

Reply:
I guess it's not easy to do so and may not be needed since the fact that experiments are Git commits is a hidden implementation detail. Will re-use this PR to roll back this change in exp-related docs instead ⌛

Reply:
Or how about adding a note like this @shcheklein ?

![image](https://user-images.githubusercontent.com/1477535/113962602-a1a63600-97ed-11eb-9f20-599bc1177695.png)

Reply:
my initial reaction - we are overthinking this a bit. How about "applies to all commits"?

Reply:
It doesn't apply to experiments (which are commits). May not be much of a confusion issue here in `fetch` but it's relevant in `gc` and `exp show/gc`. All other changes I did for consistency at this point.

Reply:
> It doesn't apply to experiments (which are commits).

I guess, I doubt anyone reading this will have a thought about experiments commits (that are hidden and far from obvious that they are commits)

It's not ideal, but trying to be too precise can even hurt since we are complicating this (force people to go and read dvc experiments and what does it at all, and if it's important for them, etc)

Reply:
OK. I removed the note everywhere except in `dvc gc` where there's actual need and some context for it. Did a bunch of other copy edits in the process 😅 PTAL